### PR TITLE
Add SwiftUI View Modifier: .animation()

### LINKED ISSUE #1568: [Term Entry] SwiftUI View Modifier: `.animation()`
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.animation()` in Swift.

A "Syntax" section with pseudo code snippet.
An "Example" section with a Swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `animation`. Then create a new file called `animation.md` under content/swiftui/concepts/viewmodifier/terms/`animation`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

Added new folder `animation` inside `content/swiftui/concepts/viewmodifier/terms`
Added new entry file `content/swiftui/concepts/terms/animation/animation.md` for SwiftUI View Modifier .animation().
Added a new animated gif for the SwiftUI View Modifier .animation() entry `swiftui-viewmodifier-animation.gif`.

Closes #1568

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/animation/animation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+The **`.animation()`** modifier applies an animation to a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+< View to be animated >
+    < View modifier that varies depending on switchValue >
+    .animation(animation, value: switchValue)
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/animation/animation.md:22]:
```suggestion
View
    .animation(animation, value: switchValue)
```

Maybe include that comment on touches on what values `switchValue` can be (include that in the second bullet point) like:

"- A `value`, when updated, triggers the animation."

File: content/swiftui/concepts/viewmodifier/terms/animation/animation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+The `.animation()` modifier takes two arguments:
+
+- An animation property or function, for example `.easeIn` or `.easeIn(duration: 2)`.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/animation/animation.md:27]: I'd love to see a list of examples here. See: https://developer.apple.com/documentation/swiftui/animation/default

and additional type properties. This does in fact include `.easeIn`, `easeOut`, `linear`, `spring`, and much more. I recommend either creating a markdown table with the type property and a description column, or a simple bulleted list with the different (bullets like shown below):
 
- .typeProperty: Followed by a description.

**Review by yangc95 — COMMENTED** [review]
Thanks for your patience @gracekishino, nice entry! Great example as well ☺️ 

I left two comments which mainly include some reworking and additional info requests.

File: content/swiftui/concepts/viewmodifier/terms/animation/animation.md (hunk 2 above)
Comment (gracekishino) [reply content/swiftui/concepts/viewmodifier/terms/animation/animation.md]: @yangc95 I added in a table or type properties and methods, thanks for the suggestion!

**Review by yangc95 — APPROVED** [review]
@gracekishino thanks for your patience! This looks good for a second review

**Review by KTom101 — APPROVED** [review]
@gracekishino I added one minor addition here: [3e0762a](https://github.com/Codecademy/docs/pull/2004/commits/3e0762a893bf744be389f26e2c8dfa9bd80cfe11). I don't see any changes needed.

**Comment by KTom101** [comment]
@gracekishino Entry is live: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/animation 🎉 Thanks for another great contribution 😄

Screenshot 2023-06-02 at 3 18 20 PM

**Comment by gracekishino** [comment]
@KTom101 thank you so much for reviewing and making the entry live!

**Pull request merged**